sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOTest.java

```diff
@@ -1142,12 +1180,20 @@ public void testWriteUnknown() throws Exception {
         .withDatasetService(datasetService);
     datasetService.createDataset("project-id", "dataset-id", "", "");
     Pipeline p = TestPipeline.create(bqOptions);
+
+    TableSchema schema = new TableSchema()
```

Comment:
<!--thread_id:cc_140597939_t; commit:7202328ef0e10895b7c80bb5d1ec39244b09c6b8; resolved:0-->
<!--section:context-quote-->
> **jkff** wrote:
> Why this change? Likewise for a couple of other changes below - they don't seem related to this PR.

<!--section:body-->
All of the TableSchema changes are indeed unneeded. I've removed them.